app/services/evaluations_api_service.rb

```
@@ -0,0 +1,69 @@
+require 'aicrowd_evaluations'
+
+class EvaluationsApiService
+  def initialize(submission_id:)
+    @submission = Submission.find(submission_id)
+    @grader_id = @submission.challenge.challenge_client_name.to_i
+  end
+
+  def call
+    grader = get_grader
+    create_submission(grader.id)
+  rescue StandardError => e
+    Submission.update(
```

We need to follow policy of failing early to make system predictable. Use `update!` here instead.